### DISCUSSION TITLE
有IOS上查看笔记的方式吗

### DISCUSSION THREAD

**Discussion by hhrensheng** [tiddly-gittly/TidGi-Desktop#466] [Q&A]
请问，我在桌面端保存的笔记请问怎么在iOS上查看？

**Comment by linonetwo** [comment]
用[TW-MobiLe-Sync手机tiddloid移动端同步TidGi桌面端插件](https://tw-cn.netlify.app/#TW-MobiLe-Sync%E6%89%8B%E6%9C%BAtiddloid%E7%A7%BB%E5%8A%A8%E7%AB%AF%E5%90%8C%E6%AD%A5TidGi%E6%A1%8C%E9%9D%A2%E7%AB%AF%E6%8F%92%E4%BB%B6)，可以直接在 Quine2 app 和太记之间同步

**Reply by hhrensheng** [reply]
感谢
